app.js

```diff
@@ -11,4 +11,152 @@ hamburger.addEventListener('click', toggleNav);
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    id: 'project3',
+    title: 'Tonic',
+    frame: ['CANOPY', 'Back End Dev', 2015],
+    tags: ['HTML', 'CSS', 'Javascript'],
+    imageUrl: '../images/feature-img1.png',
+    projectDetails:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    id: 'project4',
+    title: 'Multi-Post Stories',
+    frame: ['FACEBOOK', 'FULL STACK DEV', 2015],
+    tags: ['HTML', 'CSS', 'Javascript'],
+    imageUrl: './images/feature-img4.png',
+    projectDetails:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    liveLink: '#',
+    sourceLink: '#',
+  },
+];
```

Comment:
Good job creating an object to contain the different projects. It's would be very helpful in your future when you add the projects that you will create on the following modules! 💯 💯